### PR TITLE
Add POC of multi-date program

### LINKED ISSUE #490: Support more than two-day events in automated program - IDEA 81
<p>Yvo just sent you some feedback - and they're feeling 😀 about it! </p><p>If the program builder supported 3 (or really X) day devopsdays events. Currently when I want to define the program via the event data yml, it keys off of the first and last date, but is not happy if there is an undocumented third date. </p>

[ProdPad URL](https://portal.prodpad.com/24442/ideas/373872)

Effort: 4/10
Impact: 5/10

### PR DESCRIPTION
**Pull request by mattstratton**
The accordion is not working yet, but it seems to be fine so far.

Fixes #490

### CODE REVIEW

**Comment by mattstratton** [comment]
This also doesn't handle ignites yet...

**Comment by mattstratton** [comment]
This introduces a new regression where the accordion doesn't work properly, but I'm willing to release this with the defect since I don't think it's as important right now.

File: layouts/program/single.html [hunk 1]
```diff
@@ -1,252 +1,195 @@
...
-  {{ if ne $e.startdate $e.enddate}}
-  <div class="card">
-    <div class="card-header" role="tab" id="headingTwo">
-      <h5 class="mb-0">
-        <a data-toggle="collapse" data-parent="#accordion" href="#collapseDayTwo" aria-expanded="true" aria-controls="collapseDayTwo">
-          {{- dateFormat "January 2, 2006" $e.enddate -}}
-        </a>
-      </h5>
-    </div>
+    <!-- Get the dates for the program --->
```
Comment (kmugrage) [inline layouts/program/single.html]: extra dash in comment close

Comment (mattstratton) [inline layouts/program/single.html]: Good catch. I fixed it.

**Comment by mattstratton** [comment]
I've also tested this with a single-day event and program (hoofington) and it seems fine.

**Comment by kmugrage** [comment]
I did not build / test locally, but visual review looks good.

**Comment by bridgetkromhout** [comment]
> the accordion doesn't work properly

Or it works better than evah! If, you know, you seekritly hated the collapsing. =)

**Comment by bridgetkromhout** [comment]
This displays correctly for single and multiple-day events. I think another thing we now need is an adjustment in the program creation, which is out of scope for this PR. :shipit:

**Pull request merged**